test/api/cpp/smt2_compliance.cpp

```diff
@@ -55,15 +52,15 @@ void testGetInfo(cvc5::Solver* solver, const char* s)
 {
   std::unique_ptr<SymbolManager> symman(new SymbolManager(solver));
 
-  std::unique_ptr<Parser> p(ParserBuilder(solver, symman.get(), true).build());
-  p->setInput(Input::newStringInput(
-      "LANG_SMTLIB_V2_6", string("(get-info ") + s + ")", "<internal>"));
-  assert(p != NULL);
-  std::unique_ptr<Command> c = p->nextCommand();
-  assert(c != NULL);
-  cout << c.get() << endl;
-  stringstream ss;
+  InputParser p(solver, symman.get());
+  std::stringstream ssi;
+  ssi << "(get-info " << s << ")";
+  p.setStreamInput("LANG_SMTLIB_V2_6", ssi, "<internal>");
+  std::unique_ptr<Command> c = p.nextCommand();
+  assert(c != nullptr);
+  std::cout << c.get() << std::endl;
```

Comment:
Is this a leftover debug statement?

Reply:
I guess this is intentionally testing that the command can be printed.